build_tools/bazel/iree.bazelrc

```
@@ -338,3 +338,10 @@ build:msvc --config=msvc2019
 build:windows --config=msvc
 
 ###############################################################################
+# disable metal stuff for non Darwin platforms
+###############################################################################
+build:non_darwin --deleted_packages=//iree/hal/metal,\
+//iree/hal/metal/registration,\
+//iree/compiler/Dialect/HAL/Target/MetalSPIRV,\
+//iree/test/e2e/xla_ops/apple
```

I'd be surprised if the registration and compiler target need to be excluded here. Seems like they should build on non-mac fine since there's no objc stuff?

```suggestion
build:non_darwin --deleted_packages=//iree/hal/metal,\
//iree/test/e2e/xla_ops/apple
```